data/json/itemgroups/tools.json

```diff
@@ -224,6 +224,7 @@
       [ "pitchfork", 60 ],
       { "item": "trimmer_off", "prob": 20, "charges": [ 0, 600 ] },
       [ "scythe", 40 ],
+      [ "scythe_fake", 1 ],
```

Comment:
I'm not sure about this one, but it is an incredibly low possibility but I think this would allow it to spawn in hardware stores, which seems off.

Reply:
Removing it!